test/browser/components/side-panel/layer-list-test.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import test from 'tape';
+
+import {InitialState, StateWH3Layer} from 'test/helpers/mock-state';
+
+import * as VisStateActions from 'actions/vis-state-actions';
+import * as UIStateActions from 'actions/ui-state-actions';
+
+import {appInjector} from 'components/container';
+import {IntlWrapper, mountWithTheme} from 'test/helpers/component-utils';
+
+import LayerListFactory from 'components/side-panel/layer-panel/layer-list';
+
+const LayerList = appInjector.get(LayerListFactory);
+
+// default props from initial state
+const defaultProps = {
+  datasets: InitialState.visState.datasets,
+  layerClasses: InitialState.visState.layerClasses,
+  layerOrder: InitialState.visState.layerOrder,
+  layers: InitialState.visState.layers,
+  uiStateActions: UIStateActions,
+  visStateActions: VisStateActions
+};
+
+test('Components -> SidePanel -> LayerPanel -> LayerList -> render sortable list', t => {
+  const mock = StateWH3Layer;
+
+  let wrapper;
+  t.doesNotThrow(() => {
+    wrapper = mountWithTheme(
+      <IntlWrapper>
+        <LayerList
+          {...defaultProps}
+          isSortable
+          datasets={mock.visState.datasets}
+          layers={mock.visState.layers}
+          layerOrder={mock.visState.layerOrder}
+          layerClasses={mock.visState.layerClasses}
+        />
+      </IntlWrapper>
+    );
+  }, 'LayerList should render');
+
+  t.notEqual(wrapper.find('.sortable-layer-items').length, 0, 'should render sortable items');
```

Comment:
don't check if the number of sortable items is `0` but check the exact number you expect
```
t.equal(wrapper.find('.sortable-layer-items').length, N, 'should render N sortable items');
```

Reply:
I can do that but the idea was that 'sortable-layer-items' selector will be only present in case it is rendering the sortable list, so any number greater than zero (i was looking for a check for greaterThan, but it doesn't exist in 'tape') will mean it is a sortable list.

Reply:
yeah but we also want to make sure that the list will display exactly the items you expect, e.g. if one the layers is hidden it should not be displayed

Reply:
Shouldn't that be in a separate test? I mean, it's not the same use-case. But yeah, I'll put the exact number.